android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/mappers/PurchaseMapper.java

```diff
@@ -41,13 +41,9 @@ public PurchasesModel mapList(RequestResponse requestResponse) {
       try {
         JSONArray jsonArray = new JSONObject(response).optJSONArray("items");
         for (int i = 0; i < jsonArray.length(); i++) {
-          try {
-            JSONObject jsonObject = jsonArray.getJSONObject(i);
-            SkuPurchase skuPurchase = mapSkuPurchase(jsonObject);
-            skuPurchases.add(skuPurchase);
-          } catch (Exception e) {
```

Comment:
why did you remove the try catch?

Reply:
Read it wrong at the time, you're right, it shouldn't have been removed